docs/advanced-usage.md

```diff
@@ -0,0 +1,290 @@
+# Usage
+- [Selecting Java distribution](#Selecting-Java-distribution)
+  - [Adoptium](#Adoptium)
+  - [Zulu](#Zulu)
+- [Installing custom Java package type](#Installing-custom-Java-package-type)
+- [Installing custom Java architecture](#Installing-custom-Java-architecture)
+- [Installing custom Java distribution from local file](#Installing-Java-from-local-file)
+- [Testing against different Java distributions](#Testing-against-different-Java-distributions)
+- [Testing against different platforms](#Testing-against-different-platforms)
+- [Publishing using Apache Maven](#Publishing-using-Apache-Maven)
+- [Publishing using Gradle](#Publishing-using-Gradle)
+
+See [action.yml](../action.yml) for more details on task inputs.
+
+## Selecting Java distribution
+Input `distribution` is mandatory and should be provided to use action. See [Supported distributions](../README.md#Supported-distributions) for the list of available options.
```

Comment:
Change `and should be provided to use action` to `needs to be provided`. Also `for the list of available options` to `for a list of available options`